app/src/pages/inside/dashboardPage/dashboardList/EmptyDashboards/emptyDashboards.jsx

```diff
@@ -32,15 +40,31 @@ export class EmptyDashboards extends Component {
     intl: intlShape.isRequired,
     action: PropTypes.func,
     userDashboards: PropTypes.bool,
+    filter: PropTypes.string,
   };
 
   static defaultProps = {
     action: () => {},
     userDashboards: false,
+    filter: '',
+  };
+
+  renderNoFilteredItems = () => {
```

Comment:
Can you reuse `pages/inside/common/noResultsForFilter` component here?

Reply:
Ok. Sure. I never noticed it before.